Automattic/wp-calypso
# Site Settings: update JP Disconnect UX Survey with follow-up actions.

**Reference PR** used to bootstrap ideas. The design diverged substantially.

Complete the JP Disconnect UX Survey with the follow-up actions.

Bare-bones design demonstrating actions performed on selecting Survey Q1 for different plans. Pending design mockups.

**Depends on**:
https://github.com/Automattic/wp-calypso/pull/17962 

**To Test**:
 -  Checkout this branch, or use calypso.live
    https://calypso.live/?branch=update/site-settings-disconnect-site-survey-follow-up

For Jetpack and non-Atomic sites:
  -  verify that the options are display as shown
 
All other sites: have no access = no site visibility. 
Direct access redirects to `/settings/general:site`, where `:site` is a Jetpack site.

in progress

**Visuals**:
Free plan:

![screen shot 2017-09-10 at 14 49 19](https://user-images.githubusercontent.com/13561163/30249604-68919dc0-9637-11e7-8142-dee56b7e8540.png)

Paid plans:

![screen shot 2017-09-10 at 23 38 50](https://user-images.githubusercontent.com/13561163/30253684-f5a9db7e-9681-11e7-8d73-e5542e2214d2.png)

All plans:

![screen shot 2017-09-10 at 20 49 59](https://user-images.githubusercontent.com/13561163/30252394-9820ac20-9669-11e7-9498-d2b2aa0e0912.png)

For Premium/Pro users:

![screen shot 2017-09-10 at 14 19 52](https://user-images.githubusercontent.com/13561163/30249366-195aa39a-9633-11e7-841b-9a13d2180bad.png)
